apps/rush-lib/src/logic/test/ChangeFiles.test.ts

```diff
@@ -37,19 +44,27 @@ describe('ChangeFiles', () => {
   });
 
   describe('validate', () => {
+    it('throws when there is a patch in a hotfix branch.', () => {
+      const changeFile: string = path.join(__dirname, 'leafChange', 'change1.json');
+      const changedPackages: string[] = ['d'];
+      expect(() => {
+        ChangeFiles.validate([changeFile], changedPackages, { hotfixChangeEnabled: true } as RushConfiguration);
+      }).toThrow(Error);
+    });
+
     it('throws when there is any missing package.', () => {
       const changeFile: string = path.join(__dirname, 'verifyChanges', 'changes.json');
       const changedPackages: string[] = ['a', 'b', 'c'];
       expect(() => {
-        ChangeFiles.validate([changeFile], changedPackages);
+        ChangeFiles.validate([changeFile], changedPackages, rushConfiguration);
```

Comment:
>rushConfiguration [](start = 60, length = 17)

Why not say `{} as RushConfiguration` on each of these lines?

Reply:
In case you want to mock more stuff in the future, code reuse?